picamera2/picamera2.py

```diff
@@ -491,6 +491,10 @@ def sensor_modes(self) -> list:
                 self.sensor_modes_.append(cam_mode)
         return self.sensor_modes_
 
+    def attach_preview(self, preview) -> None:
```

Comment:
Just wondering, what happens if we have >1 preview window?

Reply:
I don't think that can happen. If we have several preview windows, it's because we have several cameras, and several Picamera2 objects with one each. So long as we're checking that there isn't already a preview when we make a new one, then I think we're OK?

Reply:
Oh gotcha, sorry forgot each preview window is for a different picamera instance.